jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -725,6 +752,7 @@ public OrderedJsonObject getDebugState(String conferenceId, String endpointId, b
         debugState.put("time", System.currentTimeMillis());
 
         debugState.put("health", getHealthStatus());
+        debugState.put("load-management", jvbLoadManager.getStats());
```

Comment:
There should probably be stats in `/colibri/stats` as well as (or possibly instead of?) `/colibri/debug`.

Reply:
I took a look and I'd like to get @bgrozev 's thoughts on an approach for this.  I think this would be suitable to be done in a follow-up PR.